### PR TITLE
Portfolio part 2

### PR DESCRIPTION
**Pull request by M1rn4**
# In this project I worked on the following activities: 
- I add more information on the HTML and CSS files 
- I used images, grid, and flexbox
- I used the linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    </section>  
+    <section class="works">
+        <div class="works_card">
+            <img class="works_img" src="./image/works_1.png" alt="works_img">
+            <h3 class="works_title">Tonic</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] I would suggest you please stick as much as possible to the Figma design. You missed `counter` a small `.` please implement it. See the image below for reference
### Figma design
 
figma
### You design
your

File: index.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            </div>
+            <a href="#" class="button">See Project</a>
+        </div>
+        <div class="works_card">
+            <img class="works_img" src="./image/works_2.png" alt="works_img">
+            <h3 class="works_title">Multi-Post<br>Stories</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] I would suggest you please stick as much as possible to the Figma design. You missed `counter` a small `.` please implement it. See the image below for reference
### Figma design
 
figma
### You design
your

File: index.html [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+            <a href="#" class="button">See Project</a>
+        </div>
+        <div class="works_card">
+            <img class="works_img" src="./image/works_3.png" alt="works_img">
+            <h3 class="works_title">Tonic</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] I would suggest you please stick as much as possible to the Figma design. You missed `counter` a small `.` please implement it. See the image below for reference
### Figma design
 
figma
### You design
your

File: index.html [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+            <a href="#" class="button">See Project</a>
+        </div>
+        <div class="works_card">
+            <img class="works_img" src="./image/works_4.png" alt="works_img">
+            <h3 class="works_title">Multi-Post<br>Stories</h3>
+            <ul class="works_options">
+                <li><a href="#" class="works_list1">CANOPY</a></li>
+                <li><a href="#" class="works_list">Back End Dev</a></li>
+                <li><a href="#" class="works_list">2015</a></li>           
+            </ul>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] I would suggest you please stick as much as possible to the Figma design. You missed `counter` a small `.` please implement it. See the image below for reference
### Figma design
 
figma
### You design
your

File: index.html [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="languages">
+                <p class="lang_skill">html</p>
+                <p class="lang_skill">css</p>
+                <p class="lang_skill">javaScript</p>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] I would suggest you please use the same background provided by microverse. See the image below for reference
### Figma design
figma
### You design
yourrrrrrrr

File: index.html [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
```
Comment (roshan-bajgain) [inline index.html]: - [x] I would suggest you please use the same background provided by microverse. See the image below for reference
### Figma design
figma
### You design
yourrrrrrrr

File: index.html [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+            </div>
```
Comment (roshan-bajgain) [inline index.html]: - [x] I would suggest you please use the same background provided by microverse. See the image below for reference
### Figma design
figma
### You design
yourrrrrrrr

File: index.html [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
```
Comment (roshan-bajgain) [inline index.html]: - [ ] I would suggest you please use the same background provided by microverse. See the image below for reference
### Figma design
figma
### You design
yourrrrrrrr

File: index.html (hunk 5 above)
Comment (roshan-bajgain) [inline index.html]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li. please implement it

File: index.html (hunk 6 above)
Comment (roshan-bajgain) [inline index.html]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li. please implement it

File: index.html (hunk 7 above)
Comment (roshan-bajgain) [inline index.html]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li. please implement it

File: index.html (hunk 8 above)
Comment (roshan-bajgain) [inline index.html]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li. please implement it

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @M1rn4, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @M1rn4, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (Kajal19-del) [inline index.html:3]: Nice uses of HTML semantic tags

File: index.html [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+                <img class="lang_arrow" src="./image/narrow.png" alt="">
+            </li>
+
+            <li class="lang_tech border">
+                <h3 class="lang_title">Skills</h3>
+                <img class="lang_arrow" src="./image/narrow.png" alt="">
+            </li>
+
+
+        </ul>
```
Comment (Kajal19-del) [inline index.html:155]: I think there should be one space line and it's looks beautiful also professional.  :100:

**Pull request merged**